GameServer.cpp

```diff
@@ -1,11 +1,13 @@
 #include "ErrorLog.h"
 #include "GameServer.h"
+#include <limits>
 
 #ifndef WIN32
 	#define closesocket close
 	#define ioctlsocket ioctl
 #endif
 
+constexpr std::size_t InvalidGameInfoIndex = (std::numeric_limits<std::size_t>::max)();

```

Comment:
MSVC requires the extra parenthesis because max is already defined (or was already defined before C++11). We could go with SIZE_MAX instead.

Reply:
I've heard a similar report on another project. The parenthesis are fine. That was the solution proposed elsewhere.